### DISCUSSION TITLE
Vim vulnerability issue on debian 11

### DISCUSSION THREAD

**Discussion by sabahsawor** [vim/vim#13427] [General]
Hi All,

Please forgive my spelling barrier.

I have Vim (version 2:8.2.2434-3+deb11u1) installed on my Debian11 server and my intrusion detection discovered there are a bunch of VCE vulnerabilities (28 VCE) with the software.
there is a reference to the issue on GitHub for example 

https://github.com/vim/vim/commit/e2bd8600b873d2cd1f9d667c28cba8b1dba18839

how to use this reference to solve the issue?

Also if there is no solution available, can I upgrade Vim to the latest version and keep running on debian 11? if yes can you help me to get this done?

I appreciate every help.

Regards,

**Comment by chrisbra** [comment]
since you are using debian, you should get fixes for thos vulnerabilities from them

**Comment by sabahsawor** [comment]
Thank you for reply.

Now I installed a new version of vim 9.0
![image](https://github.com/vim/vim/assets/121570549/2537caf7-8695-4811-9636-df820e3d9888)

the issue I have that the new vim version not showing as a package when I use  apt list --installed
see below
![image](https://github.com/vim/vim/assets/121570549/1765850e-c611-4dab-89a0-a814f5919550)

**Reply by chrisbra** [reply]
Well, if you compiled yourself, it's totally expected that it won't show up in the package managers meta-data. If you want it there, see if you can get backpots of newer Vim versions (or from 3rd-party repositories).
